app/assets/javascripts/tests/_draw_session.test.js

```diff
@@ -16,8 +16,47 @@ test('clearOtherSessions removes all markers expect the selected one', t => {
   t.end();
 });
 
+test('drawMobileSession draws a session when session is loaded and sensor is selected', t => {
+  const map = mock('drawMarker');
+  const heat = { getLevel: () => 1 };
```

Comment:
Does this influence the test output? Example if it returned 0 the output would be !=? Maybe you could make that more explicit or hide it if it's noisy